test/Microsoft.AspNetCore.Mvc.Core.Test/ModelBinding/MutableObjectModelBinderTest.cs

```diff
@@ -19,6 +21,8 @@ namespace Microsoft.AspNetCore.Mvc.ModelBinding
 {
     public class MutableObjectModelBinderTest
```

Comment:
Removed tests here that were targeted at features that have been removed. We had some tests that verify that things **don't** happen that are old features long gone from the code.